src/front/js/pages/contacts.js

```diff
@@ -0,0 +1,75 @@
+import React, { useContext } from "react";
+import "../../styles/contacts.css";
+import "bootstrap/dist/css/bootstrap.min.css";
+import Button from "react-bootstrap/Button";
+import Form from "react-bootstrap/Form";
+import { Container, Row, Col } from "react-bootstrap";
+
+const Contacts = () => {
+  return (
+    <>
+      <div className="text-center" id="titulo">
+        <h3> Our Mission</h3>
+      </div>
+      <Container className="container-fluid">
+        <Row>
+          <Col classname="col-5">
+            <img
+              src="https://images.pexels.com/photos/5691814/pexels-photo-5691814.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=650&w=940"
+              id="kids"
+            ></img>
```

Comment:
<img> is a self-closing tag:
```suggestion
            <img
              src="https://images.pexels.com/photos/5691814/pexels-photo-5691814.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=650&w=940"
              id="kids"
            />
```